src/pages/exercisePage.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head';
+
+import Layout from '../components/Layout';
+import Timer from '../components/timer/timer';
+
+import StyledExercisePage from './styledExercisePage';
+
+export default function ExercisePage() {
+	return (
+		<Layout>
+			<Head>
+				<title key="title">Exercise Configurator</title>
+				<meta key="description" name="description" content="time configurator" />
+			</Head>
```

Comment:
Shouldn't the `Head` be outside of the `Layout`?

Reply:
we recieved the template like this, so i dont really know, what to say about this. i know the head should come befor the body, but the template came like this.

Reply:
Okay 👍 :)